### PR TITLE
create order shipping UI

### PR DESCRIPTION
**Pull request by martindev727**

### CODE REVIEW

File: src/components/resource-order-shipping/resource-order-shipping.component.jsx [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+              <div className="form-row">
+                <FloatingInputLabel id="npi_number" className="input-normal" shrink={60} label="NPI Number*" onChange={(val) => this.handleInputChange('npiNumber', val)} required />
+              </div>
+              <p className="form-header">Shipping Information</p>
+              <div className="form-row">
+                <FloatingInputLabel id="address1" className="input-normal" shrink={60} label="Address1*" onChange={(val) => this.handleInputChange('address1', val)} required />
+                <FloatingInputLabel id="address2" className="input-normal" shrink={60} label="Address2" onChange={(val) => this.handleInputChange('address2', val)} />
+              </div>
+              <div className="form-row">
+                <FloatingInputLabel id="city" className="input-normal" shrink={60} label="City*" onChange={(val) => this.handleInputChange('city', val)} required />
```
Comment (rphovley) [inline src/components/resource-order-shipping/resource-order-shipping.component.jsx]: Missing bottom border on this input

screen shot 2018-09-07 at 11 36 46 am

File: src/components/resource-order-shipping/resource-order-shipping.component.jsx [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+                <FloatingInputLabel id="lastName" className="input-normal" shrink={60} label="Last Name*" onChange={(val) => this.handleInputChange('lastName', val)} required />
+              </div>
+              <div className="form-row">
+                <FloatingInputLabel id="phoneNumber" className="input-normal" shrink={60} label="Phone Number" onChange={(val) => this.handleInputChange('phoneNumber', val)} />
+                <FloatingInputLabel id="fax" className="input-normal" shrink={60} label="FAX" onChange={(val) => this.handleInputChange('fax', val)} />
+                <FloatingInputLabel id="emailAddress" type="email" className="input-large large-size" shrink={60} label="Email Address*" onChange={(val) => this.handleInputChange('emailAddress', val)} required />
+              </div>
+              <div className="form-row">
+                <FloatingInputLabel id="emailAddress" type="email" className="input-large medium-size" shrink={60} label="Email Address*" onChange={(val) => this.handleInputChange('emailAddress', val)} required />
```
Comment (rphovley) [inline src/components/resource-order-shipping/resource-order-shipping.component.jsx]: We need email validation on this field. Also, add email validation to all the forms you made in the last week or so. (Register,Update profile, etc...)

File: src/components/resource-order-shipping/resource-order-shipping.component.jsx [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+  onSubmitForm(event) {
+    event.preventDefault()
+  }
+
+  render() {
+    return (
+      <div className="resource-order-shipping">
+        <div className="primary-container">
+          <div className="page-container">
+            <h1 className="main-page-header">Shipping Information</h1>
```
Comment (rphovley) [inline src/components/resource-order-shipping/resource-order-shipping.component.jsx]: Need more space below header on desktop

screen shot 2018-09-07 at 2 10 24 pm

Line break on mobile
screen shot 2018-09-07 at 2 13 32 pm

Comment (martindev727) [reply src/components/resource-order-shipping/resource-order-shipping.component.jsx]: Fixed

File: src/components/resource-order-shipping/resource-order-shipping.component.jsx (hunk 2 above)
Comment (martindev727) [reply src/components/resource-order-shipping/resource-order-shipping.component.jsx]: Fixed

File: src/components/resource-order-shipping/resource-order-shipping.component.jsx [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+    return re.test(email);
+  }
+
+  render() {
+    return (
+      <div className="resource-order-shipping">
+        <div className="primary-container">
+          <div className="page-container">
+            <h1 className="main-page-header">Shipping Information</h1>
+            <h2 className="tertiary-header">Please enter your information below.</h2>
```
Comment (rphovley) [inline src/components/resource-order-shipping/resource-order-shipping.component.jsx]: screenshot

Some inputs still don't have bottom border

Comment (rphovley) [reply src/components/resource-order-shipping/resource-order-shipping.component.jsx]: At every page size (three column, two column, and single column) some inputs are missing a bottom border. It looks like a different input is missing the bottom border in each size

**Pull request closed without merging**